app/build.gradle.kts

```diff
@@ -0,0 +1,225 @@
+import java.util.*
```

Comment:
```suggestion
import java.util.Properties
```

Reply:
My bad, how did that go in again? Fixed the last commit